chapters/en/Unit 1 - Fundamentals/imaging.mdx

```diff
@@ -0,0 +1,29 @@
+5. Various Operations in Digital Image Processing
+In digital image processing, operations on images are diverse and can be categorized into logical, statistical, geometrical, mathematical, and transform operations. Each category encompasses specific techniques, such as Morphology under Logical Operations and Fourier and Principal Component Analysis (PCA) under Transforms. Understanding the distinction between elementwise and matrix operations is crucial in image manipulation. Elementwise operations, such as raising an image to a power or dividing it by another image, involve processing each pixel individually. This pixel-based approach contrasts with matrix operations, which utilize matrix theory for image manipulation. You can do whatever you want images as they are matrices containing numbers!
+
+6. Mathematical Tools in Image Processing
+Mathematical tools are indispensable in digital image processing. Set theory, for instance, is crucial for understanding and performing operations on images, particularly binary images. In these images, pixels are typically categorized as either foreground (1) or background (0). Operations such as union and intersection in set theory are used to determine relationships between features represented by pixel coordinates. Intensity transformations and spatial filtering are other key mathematical tools. They focus on manipulating pixel values within an image, where operators are applied to single images or a set of images for various purposes, like noise reduction.
+
+7. Spatial Filtering Techniques and Image Enhancement
+Spatial filtering encompasses a broad range of applications in image processing, primarily focused on modifying images by altering each pixel's value based on its neighbouring pixels' values. Techniques include linear spatial filters, which can either blur (lowpass filters) or sharpen (high pass filters) an image. The properties and applications of different filter kernels, such as the Gaussian filter and box filters, are contrasted. Sharpening filters, which emphasize transitions in intensity, are often implemented through digital differentiation techniques like the Laplacian, highlighting edges and discontinuities in an image.
+
+8. Image Restoration and Reconstruction Techniques
+Image restoration focuses on objectively recovering a degraded image using knowledge about the degradation phenomenon. It often involves modelling the degradation process and applying inverse processes to regain the original image. In contrast, image enhancement is more subjective, aiming to improve the visual appearance of an image. Restoration techniques include dealing with issues like noise, which can originate from various sources during image acquisition or transmission. Advanced filters, both adaptive and non-adaptive, are explored for their noise-reduction capabilities. In the field of medical imaging, particularly in computed tomography (CT), image reconstruction from projections is a crucial application.

```

Comment:
I don't think we have define what a projection is so far so it would be nice to define it here